docs/source/user-guide/cli.md

```diff
@@ -467,6 +474,66 @@ Available commands inside DataFusion CLI are:
 > SET datafusion.execution.batch_size to 1024;
 ```
 
+- `parquet_metadata` table function
```

Comment:
Here is what this looks like rendered: 
![Screenshot 2024-01-13 at 7 58 34 AM](https://github.com/apache/arrow-datafusion/assets/490673/bcedd22d-0945-4b62-bb86-dafeeb237133)